api/qan/version.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package qan;
+
+option go_package = "qan";
+
+import "google/api/annotations.proto";
+
+// Version service to check version of QAN-API.
+service Version {
+  // HandleVersion return build version of QAN-API.
+  rpc HandleVersion(VersionRequest) returns (VersionReply) {
+    option (google.api.http) = {
+      post: "/v1/qan/GetReport"
```

Comment:
Duplicate path.

Let's drop the whole service.

Reply:
I need it temporary just to initial test, remove it soon.

Reply:
Then just fix path. Right now that API can't be used:
```
2019/02/27 16:12:08 args[1:] = [qan/collector.swagger.json qan/metrics.swagger.json qan/profile.swagger.json qan/version.swagger.json]
2019/02/27 16:12:08 paths entry '/v1/qan/GetReport' already exists in primary or higher priority mixin, skipping
```

Reply:
Fixed.